### PR TITLE
Add a Makefile to automate submit-queue update and deployment for all SQs.

### PR DESCRIPTION
**Pull request by cjwagner**
This Makefile adds the ability to deploy or update all submit-queues with one make target and adds a target that updates the nginx-https-redirect and ingress used on the mungegithub cluster.
I put these targets in a new Makefile in the submit-queue directory because they only apply to the submit-queue mungegithub app.

This PR also includes a commit to improve the main mungegithub Makefile.

I could add targets that generate the nginx.conf and ingress.yaml files, but those are easy to update and updated infrequently so that might be overkill. Thoughts?

### CODE REVIEW

File: mungegithub/Makefile [hunk 1]
```diff
@@ -109,9 +109,11 @@ push_volume: volume
 ifneq ("", "$(and $(wildcard $(APP)/pv.yaml), $(wildcard $(APP)/pvc.yaml))")
 	# Creating persistent volume and persistent volume claim.
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pv.yaml
+	# Sleeping for 1s to allow persistent volume to be created.
+	@sleep 1
 	kubectl --kubeconfig=$(KUBECONFIG) create -f $(APP)/local.pvc.yaml
-	# Sleeping for 2s to allow claim to bind to volume.
-	@sleep 2
+	# Sleeping for 3s to allow claim to bind to volume.
```
Comment (fejta) [inline mungegithub/Makefile]: What is the average during? Rule of thumb is to make this value  2x-3x that value.

File: mungegithub/submit-queue/Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
+KUBECONFIG ?= $(HOME)/.kube/config
+
+# Updates nginx https proxy and the ingress with the local nginx-redirect/nginx.conf and ingress.yaml files.
+push-network:
+	# Replace the nginx config and restart the pod.
+	kubectl --kubeconfig=$(KUBECONFIG) delete cm nginx-https-redirect && \
```
Comment (fejta) [inline mungegithub/submit-queue/Makefile]: configmap and pod have better readability than cm and po. 

I recommend we use the full name for instructions and commands we check into the repo.

File: mungegithub/Makefile (hunk 1 above)
Comment (cjwagner) [inline mungegithub/Makefile]: I'm not sure how long it normally takes to bind. There was usually enough time to bind when it was set to 2 seconds, but that wasn't long enough last time I deployed. So I'm guessing 1.5 seconds as the average and changing the sleep time to 4 seconds.

File: mungegithub/submit-queue/Makefile (hunk 2 above)
Comment (cjwagner) [inline mungegithub/submit-queue/Makefile]: good point, fixed.

**Pull request merged**